apps/member/src/constants/message.ts

```diff
@@ -34,4 +34,5 @@ export const API_ERROR_MESSAGE = {
   BOOKALREADYBORROWEDEXCEPTION: '해당 도서는 대여 상태에요.',
   OVERDUEEXCEPTION: '도서 대여 연장은 최대 2회까지 가능해요.',
   NOTFOUNDEXCEPTION: '요청하신 정보를 찾을 수 없어요.',
+  ALREADYAPPLIEDEXCEPTION: '이미 신청한 활동이에요.',
```

Comment:
메시지를 하이픈으로 구분해주는 건 어떨까요? 지금 상수는 너무 읽기 어려운 것 같아요

```suggestion
  BOOK_ALREADY_BORROWED_EXCEPTION: '해당 도서는 대여 상태에요.',
  OVER_DUE_EXCEPTION: '도서 대여 연장은 최대 2회까지 가능해요.',
  NOT_FOUND_EXCEPTION: '요청하신 정보를 찾을 수 없어요.',
  ALREADY_APPLIED_EXCEPTION: '이미 신청한 활동이에요.',
```

Reply:
> 메시지를 하이픈으로 구분해주는 건 어떨까요? 지금 상수는 너무 읽기 어려운 것 같아요
> 
> 
> 
> ```suggestion
> 
>   ALREADY_BORROWED_BOOK_EXCEPTION: '해당 도서는 대여 상태에요.',
> 
>   OVER_DUE_EXCEPTION: '도서 대여 연장은 최대 2회까지 가능해요.',
> 
>   NOT_FOUND_EXCEPTION: '요청하신 정보를 찾을 수 없어요.',
> 
>   ALREADY_APPLIED_EXCEPTION: '이미 신청한 활동이에요.',
> 
> ```

해당 에러코드는 서버팀과 논의 후에 결정하게 됐어요, 해당 키가 동일하게 서버에서 리턴하여 에러 메시지와 맵핑되는 형식이에요

Reply:
@mingmingmon @limehee 에러 처리 부분 개선 가능한지 확인 부탁드려요~

Reply:
서버에서 던지는 예외를 사용자가 이해하기 어렵게 만들려는 취지에서 @gwansikk 님과 논의한 끝에 현재의 형태로 전달하게 되었어요. 서버 코드베이스에서는 예외명이 명확하게 보여 작업에 지장을 주지 않지만, 말씀해주신 것처럼 응답을 확인할 때는 이해하기 어려워 프론트엔드 입장에서 상수화하더라도 오탈자 확인이 어려워질 수 있고, 나중에 합류하는 팀원이 이 부분을 이해하는 데 어려움을 느낄 수 있다는 생각이 듭니다. 현재 서버가 Public으로 관리되고 있고, 사용자가 예외를 안다고 해서 심각한 문제가 발생할 가능성은 낮다고 판단돼요. 따라서, 프로젝트의 유지보수성과 가독성을 위해 `_`를 붙이는 방법에도 동의합니다.

다만, 가독성 문제만을 고려한다면, 서버에서 예외명을 대문자로 변환하지 않고 카멜케이스로 반환하는 방법도 고려해볼 수 있을 것 같아요. 지금처럼 `_`를 붙이는 방식은 백엔드에서 던지는 예외를 프론트엔드에서 캐치할 때 예외명을 잘못 설정하는 오류가 발생할 수 있어, 이 부분에 대한 보완이 필요할 것 같습니다.

Reply:
[989a5c1](https://github.com/kgu-clab/clab-platforms/pull/221/commits/989a5c1d215f150e195c0b1b79694b34637ad656) 아래 @gwansikk 님이 남겨주신 수정된 에러코드로 개선했어요 :)

Reply:
저도 `_`를 붙이거나 카멜케이스로 수정하는 방식에 동의합니다. 가독성을 위해서라면 개인적으로는 `_` 방식이 더 읽기 좋은데, 프론트분들이 선호하시는 방식 알려주세요!

Reply:
해당 부분은 일단 _ 없이 기존 방법으로 하는게 좋을 거 같아요. 기존 방식으로 사용하는 부분도 많고 변경점이 많아 추후 추가로 이슈를 만들어 적용하는 것으로 해요. 관련해서 변경해야할 시기가 되면 알려드릴게요.

@Jeong-Ag @SWARVY @limehee @mingmingmon

Reply:
네 확인했습니다~